sdk/appconfiguration/azure-appconfiguration/CHANGELOG.md

```diff
@@ -1,16 +1,14 @@
 # Release History
 
-## 1.5.1 (Unreleased)
+## 1.5.1 (2024-03-07)
 
 ### Features Added
-
-### Breaking Changes
+- Exposed `send_request()` method in each client to send custom requests using the client's existing pipeline.
```

Comment:
Do we have samples demo how to use `send_request` method? If not, please add some.

Reply:
Not yet, we are using `send_request` in the same test with etag. I agree to add specific samples for `send_request`.

Reply:
Added samples.